src/qililab/instruments/qdevil/qdevil_qdac2.py

```diff
@@ -35,6 +35,8 @@ class QDevilQDac2(VoltageSource):
         settings (QDevilQDac2Settings): Settings of the instrument.
     """
 
+    _MIN_QDAC: float = 0.01
+    _MAX_QDAC: float = 2e7
```

Comment:
```suggestion
    _MIN_RAMPING_RATE: float = 0.01
    _MAX_RAMPING_RATE: float = 2e7
```

Could you rename those two?